src/sentry/tasks/low_priority_symbolication.py

```diff
@@ -0,0 +1,87 @@
+"""
+Tasks that automate the job of moving projects in and out of symbolicator's low priority queue based
+on symbolication metrics stored in Redis.
+
+This has three major tasks, executed in the following general order:
+1. Scan for new suspect projects in Redis that need to be checked for LPQ eligibility. Triggers 2 and 3.
+2. Determine a project's eligibility for the LPQ based on their recorded metrics.
+3. Remove some specified project from the LPQ.
+"""
+
+import logging
+
+import sentry_sdk
+
+from sentry.processing.realtime_metrics import realtime_metrics_store
+from sentry.processing.realtime_metrics.base import BucketedCount
+from sentry.tasks.base import instrumented_task
+
+logger = logging.getLogger(__name__)
+
+
+@instrumented_task(
+    name="sentry.tasks.symbolicator.scan_for_suspect_projects",
+    queue="symbolicator.compute_low_priority_queue",
+    ignore_result=True,
+    soft_time_limit=10,
+)  # type: ignore
+def scan_for_suspect_projects() -> None:
+    _scan_for_suspect_projects()
+
+
+def _scan_for_suspect_projects() -> None:
+    suspect_projects = set()
+    for project_id in realtime_metrics_store.get_lpq_candidates():
+        suspect_projects.add(project_id)
+        compute_lpq_eligibility(project_id)
+
+    current_lpq_projects = realtime_metrics_store.get_lpq_projects() or set()
+    deleted_projects = current_lpq_projects.difference(suspect_projects)
+
+    if len(deleted_projects) == 0:
+        return
+
+    removed = realtime_metrics_store.remove_projects_from_lpq(deleted_projects)
+
+    # TODO: should this just be logger.warning(...)?
+    if len(removed) > 0:
+        sentry_sdk.capture_message(
+            f"Moved project(s) {removed} out of symbolicator's low priority queue.", level="info"
+        )
```

Comment:
why is this worthy of a sentry event at all?

Reply:
i think if moving in to the LPQ is considered a notable event, then moving out should be as well. if sentry events end up being the only thing that leaves a breadcrumb trail about a project's membership in the queue, then knowing when it leaves is useful so we can determine how long a project was in the LPQ. 

we can derive other useful metrics with this info, such as how long on average projects tend to stay in the LPQ, and potentially if they tend to exhibit LPQ-worthy behaviour during specific times of the day. i could be getting a little ahead of myself here though. thoughts?